coinmarketcap/core.py

```diff
@@ -2,23 +2,27 @@
 # -*- coding: utf-8 -*-
 
 import json
+import os
 import requests
 import requests_cache
+import tempfile
 
 class Market(object):
 
 	_session = None
 	__DEFAULT_BASE_URL = 'https://api.coinmarketcap.com/v1/'
 	__DEFAULT_TIMEOUT = 120
 
-	def __init__(self, base_url = __DEFAULT_BASE_URL, request_timeout = __DEFAULT_TIMEOUT):
+	def __init__(self, base_url = __DEFAULT_BASE_URL, request_timeout = __DEFAULT_TIMEOUT, tempdir_cache = False):
```

Comment:
Could you please create a global __TEMPDIR_CACHE variable? Makes it easier to read all the vars at one glance.